packages/ethereum/test/HoprChannels.spec.ts

```diff
@@ -53,6 +54,47 @@ export const validateChannel = (actual, expected) => {
   expect(actual.status.toString()).to.equal(expected.status)
 }
 
+type ChannelArrProps = [BigNumber, string, BigNumber, BigNumber, number, BigNumber, number]
```

Comment:
have u tried to get these types from typechain?

Reply:
Yeah, sadly they don't have a type...

```typescript
    channels(
      arg0: BytesLike,
      overrides?: CallOverrides
    ): Promise<
      [BigNumber, string, BigNumber, BigNumber, number, BigNumber, number] & {
        balance: BigNumber;
        commitment: string;
        ticketEpoch: BigNumber;
        ticketIndex: BigNumber;
        status: number;
        channelEpoch: BigNumber;
        closureTime: number;
      }
    >;
```